edocumentation-service/src/main/java/org/dcsa/edocumentation/service/ActiveReeferSettingsService.java

```diff
@@ -0,0 +1,34 @@
+package org.dcsa.edocumentation.service;
+
+import lombok.AllArgsConstructor;
+import org.dcsa.edocumentation.domain.persistence.entity.ActiveReeferSettings;
+import org.dcsa.edocumentation.domain.persistence.repository.ActiveReeferSettingsRepository;
+import org.dcsa.edocumentation.service.mapping.ActiveReeferSettingsMapper;
+import org.dcsa.edocumentation.transferobjects.BookingActiveReeferSettingsTO;
+import org.dcsa.edocumentation.transferobjects.ShippingInstructionActiveReeferSettingsTO;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+
+@Service
+@AllArgsConstructor
+public class ActiveReeferSettingsService {
+  private final ActiveReeferSettingsRepository activeReeferSettingsRepository;
+  private final ActiveReeferSettingsMapper mapper;
+
+  @Transactional(Transactional.TxType.MANDATORY)
```

Comment:
I thought we (the dev team) agreed we'd use the spring transactional over the javax one to keep consistency, so this would need to change.

Reply:
we are using this annotation everywhere